src/BinaryParsers/PEBinary/ProgramDatabase/Pdb.cs

```diff
@@ -259,7 +262,7 @@
 
         private readonly Lazy<HashSet<uint>> writableSegmentIds;
 
-        private HashSet<uint> GenerateWritableSegmentSet()
+        public HashSet<uint> GenerateWritableSegmentSet()

```

Comment:
>public [](http://example.com/codeflow?start=8&length=6)

change to internal, since this only gets called by the constructors and the customer/us would not need to manually invoke this.